app/src/main/java/com/teamzzong/hacker/presentation/home/UserDetailViewModel.kt

```diff
@@ -0,0 +1,28 @@
+package com.teamzzong.hacker.presentation.home
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.teamzzong.hacker.domain.entity.user.DetailProfile
+import com.teamzzong.hacker.domain.repository.user.UserRepository
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
+import timber.log.Timber
+import javax.inject.Inject
+
+@HiltViewModel
+class UserDetailViewModel @Inject constructor(
+    private val repository: UserRepository
+) : ViewModel() {
+    private val _userDetailState = MutableStateFlow<DetailProfile?>(null)
+    val userDetailState = _userDetailState.asStateFlow()
+
+    fun setUserDetailScreen() {
+        viewModelScope.launch {
+            kotlin.runCatching { repository.getMyDetailProfile() }
```

Comment:
```suggestion
            runCatching { repository.getMyDetailProfile() }
```

그냥 kotlin 빼도 되는데 이건 취향차이여서

Reply:
취향 반영했습니다. ^_^